test/CappedFallbackRateSource.t.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { RateSourceMock } from "./mocks/RateSourceMock.sol";
+
+import { CappedFallbackRateSource } from "../src/CappedFallbackRateSource.sol";
+
+contract RevertingRateSource {
+    function getAPR() external pure returns (uint256) {
+        revert("RevertingRateSource/some-error");
+    }
+}
+
+contract CappedFallbackRateSourceTest is Test {
+
+    RateSourceMock originalSource;
+
+    CappedFallbackRateSource rateSource;
+
+    function setUp() public {
+        originalSource = new RateSourceMock(0.037e18, 18);
+
+        rateSource = new CappedFallbackRateSource({
+            _source:      address(originalSource),
+            _lowerBound:  0.01e18,
+            _upperBound:  0.08e18,
+            _defaultRate: 0.03e18
+        });
+    }
+
+    function test_constructor() public {
+        assertEq(address(rateSource.source()), address(originalSource));
+        assertEq(rateSource.lowerBound(),      0.01e18);
+        assertEq(rateSource.upperBound(),      0.08e18);
+        assertEq(rateSource.defaultRate(),     0.03e18);
+        assertEq(rateSource.decimals(),        18);
+    }
+
+    function test_rate_within_bounds() public {
```

Comment:
Weird casing in these tests
`test_rate_within_bounds` -> `test_rateWithinBounds`
Applicable to all in the file